Astro 1284 update image assets for astro uxds

Design-Kat opened the pull request:
https://rocketcom.atlassian.net/browse/ASTRO-1284

Should be all the currently updated Image Assets for PI 2021-1.

@amalyah please review when you get a chance to ensure I've included all the correct images. Not included: Any image from service-specific folders, and images blocked out as still needing update/links to Astro 6.

Design-Kat commented:
Note: All images should have been updated that need updating at this point (previously Sign-in was still being completed).

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -16,10 +16,12 @@ theme: true
+ When implementing validation and security measures Astro users should consult with their organization's IT/Security departments and adapt these patterns to suit the requirements of their organization’s best practices and security needs.
  :::
 
+[Example of a full-page, simple sign in screen](/img/components/signin-simple-full.png "Example of a full-page, simple sign in screen")
```
Please add an exclamation point (!) here to get the image to render e.g. 

```
![Example of a full-page, simple sign in screen](/img/components/signin-simple-full.png "Example of a full-page, simple sign in screen")
```

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
 
 ### Sign in with Username or Email Plus Password
 
+[Example of a sign in screen hosted within a modal dialog](/img/components/signin-dialog-simple.png "Example of a sign in screen hosted within a modal dialog")
```
Exclamation point

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
 
+### Sign in with Single Sign-on (SSO)
+
+[Example of Single Sign-on hosted within a modal dialog](/img/components/signin-dialog-sso.png "Example of Single Sign-on hosted within a modal dialog")
```
Please add exclamation point before the opening bracket

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
 
 The sign in UI elements can be added in 3 possible placement areas.
 
+[Example of a sign in screen hosted within a modal dialog](/img/components/signin-dialog-simple.png "Example of a sign in screen hosted within a modal dialog")
```
Please add exclamation point before the opening bracket

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
 * Can allow application to be running in the background when signed out
 * Ideal for switching applications
 
+[Example of a sign in screen located within a sidebar](/img/components/signin-sidebar-simple-full.png "Example of a sign in screen located within a sidebar")
```
Please add exclamation point before the opening bracket

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
 * Allows imagery/branding on opposite side of page to differentiate application
 * Often used alongside SSO
 
+[Example of a full-page sign in screen](/img/components/signin-simple-full.png "Example of a full-page sign in screen")
```
Please add exclamation point before the opening bracket

dmcalester commented on _content/components/sign-in.md:
```diff
@@ -88,31 +102,33 @@ Common server-side errors:
 * Account lock out
 * Database or system error
 
+[Example of a full-page account lock out (access denied) screen](/img/components/signin-accessdenied-full.png "Example of a full-page account lock out (access denied) screen")
```
Please add exclamation point before the opening bracket

dmcalester reviewed (changes requested):
See inline comments all reference adding a missing exclamation point to some images so they render. Otherwise approved.

Design-Kat commented:
> See inline comments all reference adding a missing exclamation point to some images so they render. Otherwise approved.

Should be all in now, thank you, @dmcalester !

amalyah reviewed (changes requested):
- Design Guidelines/Grid images look blurry. If we add them to the re-export, could that be fixed?
- Gradual Visualization Palette image updates needs to be removes or they need to be re-exported without the NEEDS TO BE LINKED text and opacity cover over them.
- Help text do and dont images aren't showing up on the Forms and Validation page.
- New checkbox images aren't showing up on the checkbox page
- Sign In large images aren't showing up on the Sign In page.
- Some images changed size slightly, others by a lot. Not sure if that's an issue or not.

Design-Kat commented:
@amalyah 

* Design Guidelines/Grid images look blurry: This is because it’s a bitmap, the original needs to be found and added- maybe that should be a separate ticket/bug? 
*  Gradual Visualization Palette: I re-exported them without the opacity over it.
* Fixed some folder and image name issues on Checkbox, sign in, and forms & validation that should fix the issues you saw with images showing up.

dmcalester reviewed (changes requested):
The full screen images for modal, sidebar and full page are mis-rendering @nortonprojects could you lend a hand?

amalyah reviewed (changes requested):
Only thing that I see wrong now is the Sign In images that Duncan already mentioned.

amalyah reviewed (approved):
Styling issue: Numbered bullet points do not keep the numbers that Kat entered in the code.

dmcalester reviewed (approved):
Approved for image update.

amalyah reviewed (approved):
Numbering issue is fixed now too.

nortonprojects commented:
> Numbering issue is fixed now too.

Touching up the sizing, and I'll clean up the git history a bit

Design-Kat commented:
@dmcalester @Cheshire89 it looks like this has all the approvals required as of yesterday, could this be merged so the ticket can be closed?
